app/src/main/java/com/framgia/fsalon/data/source/api/FSalonApi.java

```diff
@@ -1,5 +1,7 @@
 package com.framgia.fsalon.data.source.api;
 import com.framgia.fsalon.data.model.BillResponse;
+import com.framgia.fsalon.data.model.BookingCustomer;
```

Comment:
ERROR: Unused import: com.framgia.fsalon.data.model.BookingCustomer. Use Ctrl+Shift+O (Eclipse) or Ctrl+Alt+O (Intellij) to sort imports